src/components/navigation-header/navigation-header.module.css

```diff
@@ -68,11 +68,14 @@
   align-items: center;
   display: flex;
   justify-content: left;
-  width: calc(
-    var(--dev-dot-sidebar-width) + var(--main-area-padding-left) -
-      var(--header-padding-left-right) -
-      var(--header-dropdown-menu-padding-left-right)
-  );
+
+  @media (--dev-dot-desktop) {
```

Comment:
Seems like we only want to run this specific calculation when we have the desktop nav layout. Seemed like the easiest solution here, open to input!

Reply:
This is great! Exactly what's need 🙌🏻